### PR TITLE
Jwt another version bramberry

### PR DESCRIPTION
**Pull request by bramberry**

### CODE REVIEW

File: src/main/postman/jersey-jwt-springsecurity.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,272 @@
+{
```
Comment (DenSev) [inline src/main/postman/jersey-jwt-springsecurity.postman_collection.json]: Delete this.

File: src/main/postman/jersey-jwt-springsecurity.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (DenSev) [inline src/main/postman/jersey-jwt-springsecurity.postman_environment.json]: Delete this.

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+server:
```
Comment (DenSev) [inline src/main/resources/application.yaml]: Merge application.properties and application.yaml into a single file. application.yaml would probably be better option.

File: src/main/postman/jersey-jwt-springsecurity.postman_collection.json (hunk 1 above)
Comment (bramberry) [reply src/main/postman/jersey-jwt-springsecurity.postman_collection.json]: deleted

File: src/main/postman/jersey-jwt-springsecurity.postman_environment.json (hunk 2 above)
Comment (bramberry) [reply src/main/postman/jersey-jwt-springsecurity.postman_environment.json]: deleted

File: src/main/resources/application.yaml (hunk 3 above)
Comment (bramberry) [reply src/main/resources/application.yaml]: done

**Pull request merged**